src/AutorestSwift/Models/Complex/ArraySchema.swift

```diff
@@ -28,7 +28,7 @@ import Foundation
 
 class ArraySchema: Schema {
     /// elementType of the array
-    let elementType: Schema
+    let elementType: Schema?
```

Comment:
Shouldn't ArraySchema always have a elementType? Why `elementType` changes to optional here?

Reply:
Same reason as Schema in the Value class.

Reply:
I was able to revert this back to being required without issue.